### PR TITLE
Remove user data from localStorage after logout

### PR DESCRIPTION
**Pull request by sonmezali**

### CODE REVIEW

File: client/src/components/ApplicantProfile.js [hunk 1]
```diff
@@ -70,8 +70,14 @@ class ApplicantProfile extends React.Component {
   // Get data from db
   componentWillMount() {
     const loggedInUser = JSON.parse(localStorage.getItem("user"));
-    const userId = loggedInUser.user_id;
-    this.setState({ userId: userId });
+    if (loggedInUser) {
+      const userId = loggedInUser.user_id;
+      this.setState({ userId: userId });
+      console.log(loggedInUser);
```
[the remark this answers is not in the document]
Comment (sonmezali) [reply client/src/components/ApplicantProfile.js]: It is done

File: client/src/components/ApplicantProfile.js [hunk 2]
```diff
@@ -70,8 +70,14 @@ class ApplicantProfile extends React.Component {
...
+    } else {
+      // window.location.replace("/applicant-profile");
```
[the remark this answers is not in the document]
Comment (sonmezali) [reply client/src/components/ApplicantProfile.js]: done

File: client/src/components/SideBar.js [hunk 3]
```diff
@@ -6,8 +6,10 @@ export default ({
   visible,
   activeItem,
   handleSidebarHide,
-  handleItemClick
+  handleItemClick,
+  user
 }) => {
+  console.log(user);
```
[the remark this answers is not in the document]
Comment (sonmezali) [reply client/src/components/SideBar.js]: done

**Comment by youssef910** [comment]
@sonmezali I have changed  a lot  of things at the way you get the user data and save it also I did the part of deleting the user data from local storage

**Pull request closed without merging**